audb/core/api.py

```diff
@@ -442,7 +442,7 @@ def remove_media(
         name: str,
         files: typing.Union[str, typing.Sequence[str]],
         *,
-        verbose: bool = False,
+        verbose: bool = True,
```

Comment:
Mhh, not so sure if we should really change this. @hagenw what do you think?

Reply:
Actually reverted to `False` to not break the API.

Reply:
Agree.